test_package.go

```diff
@@ -82,9 +83,30 @@ func (tp *TestPackage) Cleanup() {
 // It will construct the blueprint if it doesn't already exist in the docker image cache.
 // This function is the main setup function for all tests as it provides a deployment with
 // which tests can interact with.
-func (tp *TestPackage) Deploy(t *testing.T, blueprint b.Blueprint) Deployment {
+func (tp *TestPackage) OldDeploy(t *testing.T, blueprint b.Blueprint) Deployment {
 	t.Helper()
 	timeStartBlueprint := time.Now()
+	if err := tp.complementBuilder.ConstructBlueprintIfNotExist(blueprint); err != nil {
+		t.Fatalf("OldDeploy: Failed to construct blueprint: %s", err)
+	}
+	namespace := fmt.Sprintf("%d", atomic.AddUint64(&tp.namespaceCounter, 1))
+	d, err := docker.NewDeployer(namespace, tp.complementBuilder.Config)
+	if err != nil {
+		t.Fatalf("OldDeploy: NewDeployer returned error %s", err)
+	}
+	timeStartDeploy := time.Now()
+	dep, err := d.Deploy(context.Background(), blueprint.Name)
+	if err != nil {
+		t.Fatalf("OldDeploy: Deploy returned error %s", err)
+	}
+	t.Logf("OldDeploy times: %v blueprints, %v containers", timeStartDeploy.Sub(timeStartBlueprint), time.Since(timeStartDeploy))
+	return dep
+}
+
+func (tp *TestPackage) Deploy(t *testing.T, numServers int) Deployment {
```

Comment:
If I got that correctly, the new Deploy only cares about spawning N servers, and does not preconfigure users on it?

Reply:
Correct.